snippets/mcs-endpoint-limitation.adoc

```diff
@@ -0,0 +1,11 @@
+// Text snippet included in the following modules:
+//
+// * modules/about-crio.adoc
+// * modules/nodes-containers-using.adoc
+
+:_content-type: SNIPPET
+
+[IMPORTANT]
+====
+Due to a known issue, it is recommended to use OpenShift SDN to secure the machine config server (MCS) endpoints: ports 22623 and 22624. Also, because of this issue, adding secrets to the ignition configuration file is not supported.
```

Comment:
"adding secrets to the ignition configuration file is not supported" or should it be more like "is strongly discouraged"?